code/test/util.js

```diff
@@ -215,6 +269,123 @@ const getEOTP = async ({ seed, hseed, effectiveTime, timeOffset }) => {
   return { index, eotp }
 }
 
+// transactionExecute commits and reveals a wallet transaction
+const transactionExecute = async ({ wallet, operationType, tokenType, contractAddress, tokenId, dest, amount, data, address, randomSeed, backlinkAddresses, testTime }) => {
```

Comment:
I suggest breaking down this function and only  let it handle general, frequently occurring cases. Transaction execution functions for infrequent operations, if needed, can live in their own files. People testing individual functions shouldn't need to come to this centralized function and going over the entire thing to understand what's needed

Reply:
Good Point 🙏 
Not sure the best way to do this.
I've added this to a doc to keep track of these items [REFACTOR.md](https://github.com/1wallet-dev/1wallet/blob/testing/code/test/REFACTOR.md)